.github/workflows/ci-helm-workflow.yml

```diff
@@ -0,0 +1,42 @@
+name: Continuous Integration - Helm
+
+on:
+  push:
+    branches: [ master ]
+    paths:
+    - 'aio/deploy/helm-chart/**'
+  pull_request:
+    paths:
+    - 'aio/deploy/helm-chart/**'
+
+jobs:
+  helm-lint-test:
```

Comment:
nit: Can you name the job? Similar to how it's done for other jobs: https://github.com/kubernetes/dashboard/blob/665b4d367d61dc429d3765982ced3c4aab08daa7/.github/workflows/ci-workflow.yml#L9-L12

Reply:
Done!